### ISSUE TITLE
CLI options definitions were upgraded with "type" property

### ISSUE THREAD

**Issue by hrvacadaIM** [raisenational/serverless-dynamodb#50]
I am using this plugin and I am getting this warning with serverless v3.38.0:

```
CLI options definitions were upgraded with "type" property (which could be one of "string", "boolean", "multiple"). Below listed plugins do not predefine type for introduced options:
 - ServerlessDynamoDBPlugin for "seed"
```

I see that you left out type intentionally https://github.com/raisenational/serverless-dynamodb/blob/master/src/index.ts#L59, but with serverless v4.0.0, this is going to be an error: 

```
Starting with v4.0.0 any option extensions which does not have type defined will be communicated with a thrown error
```

You could maybe use type `multiple`?

**Comment by domdomegg** [comment]
Whoops, thought we fixed this in #24 but ended up missing this one. Good catch, will fix this one too.

**Comment by domdomegg** [comment]
Fixed in v0.2.53
